main.go

```diff
@@ -87,6 +87,10 @@ func main() {
 		rest.HandleActivityTick(c)
 	})
 
+	r.GET("/healthz", func(c *gin.Context) {
+		c.Writer.WriteHeader(http.StatusOK)
```

Comment:
I wonder if it would a bit better to return NoContent here.

Reply:
NoContent is usually for POST/PUT, I don't know if it applies very well to a GET